homeassistant/components/mystrom/light.py

```diff
@@ -89,7 +100,7 @@ def __init__(self, bulb, name, mac):
     @property
     def name(self):
         """Return the display name of this light."""
-        return self._name
+        return self._mac
```

Comment:
You probably think I'm being a pain... but I think in a preliminary PR you should move most of light and switch properties to attributes in `__init__` method.
For example:
```python
self._attr_name = name
self._attr_unique_id = mac
self._attr_brightness = ...
self._attr_hs_color = ...
self._attr_available = ...
self._attr_effect_list = ...
self._attr_is_on = ...
```

Reply:
I submitted the PR #93591